### PR TITLE
ci action template

### PR DESCRIPTION
**Pull request by kumarabd**
Signed-off-by: kumarabd <abishekkumar92@gmail.com>

**Description**

- Build and release action for Getnighthawk release
- Action template and scripts initialised.

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: .github/workflows/build-and-publish.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+name: Build and Publish Assets
+on: 
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version to be released'
+        required: true
+        default: 'stale'
```
Comment (leecalcote) [inline .github/workflows/build-and-publish.yaml:8]: Stale or "stable"?

File: .github/workflows/build-and-publish.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+jobs:
+  release:
+    name: create release
+    runs-on: ubuntu-latest
+    steps:
+    - name: Create Release
+      id: create_release
+      uses: actions/create-release@v1
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} # This token is provided by Actions, you do not need to create your own token
```
Comment (leecalcote) [inline .github/workflows/build-and-publish.yaml:19]: May need to be GH_ACCESS_TOKEN

Comment (kumarabd) [reply .github/workflows/build-and-publish.yaml:19]: okay

File: .github/workflows/build-and-publish.yaml (hunk 1 above)
Comment (kumarabd) [reply .github/workflows/build-and-publish.yaml:8]: version is mandatory, so stale is like an invalid version

Comment (kumarabd) [reply .github/workflows/build-and-publish.yaml]: made it stable now.

**Pull request merged**